connect/mirror/README.md

```diff
@@ -195,8 +195,46 @@ target cluster. In most cases, your entire organization should use a single
 MM2 configuration file.
 
 ### Exactly-once
-Exactly-once delivery is not currently supported for dedicated MM2 clusters. In order to run MM2 with exactly-once, deploy it onto a
-Connect distributed cluster that is configured with exactly-once support enabled.
+Exactly-once semantics are supported for dedicated MM2 clusters as of version 3.5.0.
+
+For new MM2 clusters, set the `exactly.once.source.support` property to `enabled` for
+all targeted Kafka clusters that should be written to with exactly-once semantics. For example,
+to enable exactly-once for writes to cluster `B`, add the following to your MM2 config file:
+
+    B.exactly.once.source.support = enabled
+
+For existing MM2 clusters, a two-step upgrade is necessary. Instead of immediately
+setting the `exactly.once.source.support` property to `enabled`, first set it to `preparing` on
+all nodes in the cluster. Once this is complete, it can be set to `enabled` on all nodes in the
+cluster, in a second round of restarts.
+
+In either case, it is also necessary to enable intra-cluster communication between your MM2
+nodes, as described in
+[KIP-710](https://cwiki.apache.org/confluence/display/KAFKA/KIP-710%3A+Full+support+for+distributed+mode+in+dedicated+MirrorMaker+2.0+clusters).
+To do this, the `dedicated.mode.enable.internal.rest` property must be set to `true` in your MM2 config.
+In addition, many of the REST-related
+[configuration properties available for Kafka Connect](https://kafka.apache.org/documentation/#connectconfigs)
+can be specified in your MM2 config. For example, to enable intra-cluster communication in your MM2
+cluster with each node listening on port 8080 of their local machine, add this to your config file:
+
+    dedicated.mode.enable.internal.rest = true
+    listeners = http://localhost:8080
+
+**Note that, if intra-cluster communication is enabled in production environments, it is highly
+recommended to secure the REST servers brought up by each MM2 node. See the configuration
+properties for Kafka Connect for information on how this can be accomplished.**
+
+It is also recommended to filter records from aborted transactions out from replicated data
```

Comment:
Should `recommended` be `required`? as you can't enable EoS without setting `isolation.level=read_committed`